try-it.md

```diff
@@ -32,10 +32,12 @@ permalink: /try-it.html
 
 ### 1.1. Prerequisites
 
-- System with CentOS 8 or Ubuntu 18.04
+- System with CentOS 8 or Ubuntu 20.04
 - Bare metal preferred, as we will be creating VMs to emulate bare metal hosts
 - Run as a user with passwordless sudo access
-- Minimum resource requirements for the host machine: 4C CPUs, 16 GB RAM memory.
+- Minimum resource requirements for the host machine: 4C CPUs, 16 GB RAM memory
+- The host machine should have a floating IP
```

Comment:
I think this is not necessary but please correct me if I'm wrong. It should be possible to run all of this locally without any floating IP.